cmd/cdi-cloner/clone-source.go

```diff
@@ -199,10 +204,11 @@ func main() {
 	serverCert := []byte(getEnvVarOrDie("SERVER_CA_CERT"))
 
 	url := getEnvVarOrDie("UPLOAD_URL")
+	preallocation, _ := strconv.ParseBool(getEnvVarOrDie(common.Preallocation)) // False is default in case of error
```

Comment:
maybe if there was an error and it uses false as default should print some warning or something to explain that the default  false is used.

Reply:
Done